src/components/LogEditForm.js

```diff
@@ -0,0 +1,101 @@
+import { useState, useEffect } from "react";
+import { useParams, Link, useNavigate } from "react-router-dom";
+import axios from "axios";
+
+function LogEditForm() {
+  let { index } = useParams();
+
+  const [log, setLog] = useState({
+    name: "",
```

Comment:
this key does not match what you have on line 55, which is causing the app to throw warnings. it also doesn't match what you have in your backend in models